{
  "Faults": [
    {
      "latitude": 51.479565871658906,
      "type": "LV",
      "reference": "KU7763",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T15:00:00.000+0000",
      "affectedCustomerCount": 25,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a6b1e7b5-8c8e-4fe3-af8f-ab1d0d5dd2ce",
      "minLongitude": -0.5854,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5791277497060789,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T14:19:00.000+0000",
      "estimatedRestoration": "2023-04-23T00:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7763'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7763",
      "maxLongitude": -0.5712,
      "minLatitude": 51.4727,
      "name": "SL3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL3 9BU",
        "SL3 9BY",
        "SL3 9BZ",
        "SL3 9DB",
        "SL3 9DD",
        "SL3 9DE",
        "SL3 9DQ",
        "SL3 9EQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5854,
              51.4836
            ],
            [
              -0.5712,
              51.4836
            ],
            [
              -0.5712,
              51.4727
            ],
            [
              -0.5854,
              51.4727
            ],
            [
              -0.5854,
              51.4836
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-22T15:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4836,
      "etrUpdate": true
    },
    {
      "latitude": 50.845396772249735,
      "type": "HV",
      "reference": "KU7820",
      "etaUpdate": true,
      "affectedCustomerCount": 103,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "570a5db6-05d5-4063-9dd3-298741545e32",
      "minLongitude": -0.9539,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9262416750715681,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T17:00:00.000+0000",
      "estimatedRestoration": "2023-04-22T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7820'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7820",
      "maxLongitude": -0.9025,
      "minLatitude": 50.828,
      "name": "PO10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO10 7AG",
        "PO10 7AL",
        "PO10 7AN",
        "PO10 7AQ",
        "PO10 7AT",
        "PO10 7AX",
        "PO10 7AY",
        "PO10 7AZ",
        "PO10 7BD",
        "PO10 7BH",
        "PO10 7BJ",
        "PO10 7BQ",
        "PO10 7BS",
        "PO10 7EG",
        "PO10 7EH",
        "PO10 7EL",
        "PO10 7EN",
        "PO10 7EQ",
        "PO10 7JQ",
        "PO10 7NA",
        "PO10 8AN",
        "PO10 8AP",
        "PO10 8AR",
        "PO10 8AS",
        "PO10 8AT",
        "PO10 8AU",
        "PO10 8AW",
        "PO10 8AX",
        "PO10 8AY",
        "PO10 8AZ",
        "PO10 8BA",
        "PO10 8BB",
        "PO10 8BD",
        "PO10 8BE",
        "PO10 8BF",
        "PO10 8BG",
        "PO10 8BH",
        "PO10 8BJ",
        "PO10 8BL",
        "PO10 8BN",
        "PO10 8BP",
        "PO10 8BQ",
        "PO10 8BS",
        "PO10 8BT",
        "PO10 8BU",
        "PO10 8BW",
        "PO10 8BZ",
        "PO10 8DB",
        "PO10 8DD",
        "PO10 8DG",
        "PO10 8DQ",
        "PO10 8EA",
        "PO10 8ET",
        "PO10 8EU",
        "PO10 8EX",
        "PO10 8EZ",
        "PO10 8FQ",
        "PO10 8FS",
        "PO10 8H",
        "PO10 8HA",
        "PO10 8HE",
        "PO10 8HF",
        "PO10 8HJ",
        "PO10 8HR",
        "PO10 8HS",
        "PO10 8HT",
        "PO10 8HU",
        "PO10 8HX",
        "PO10 8JB",
        "PO10 8JD",
        "PO10 8JJ",
        "PO10 8JL",
        "PO10 8JN",
        "PO10 8JR",
        "PO10 8JY",
        "PO10 8NY",
        "PO10 8UE",
        "PO10 8XD",
        "PO10 8XT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9539,
              50.8621
            ],
            [
              -0.9025,
              50.8621
            ],
            [
              -0.9025,
              50.828
            ],
            [
              -0.9539,
              50.828
            ],
            [
              -0.9539,
              50.8621
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-22T17:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "b0ae007c-639d-44bc-ad9e-3d8c768eca48"
      ],
      "maxLatitude": 50.8621,
      "etrUpdate": true
    },
    {
      "latitude": 50.83952010481921,
      "type": "LV",
      "reference": "KU7791",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T16:45:00.000+0000",
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "cdf5b7b9-b761-4c3a-a2ed-dce88e8e6be0",
      "minLongitude": -0.7377,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7280679031135138,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T15:48:00.000+0000",
      "estimatedRestoration": "2023-04-22T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7791'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7791",
      "maxLongitude": -0.7198,
      "minLatitude": 50.8352,
      "name": "PO20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO20 2BP",
        "PO20 2EU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7377,
              50.8506
            ],
            [
              -0.7198,
              50.8506
            ],
            [
              -0.7198,
              50.8352
            ],
            [
              -0.7377,
              50.8352
            ],
            [
              -0.7377,
              50.8506
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-22T17:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8506,
      "etrUpdate": true
    },
    {
      "latitude": 50.86956320807633,
      "type": "HV",
      "reference": "KU7847",
      "etaUpdate": true,
      "affectedCustomerCount": 382,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a80cb74e-6329-46e5-8192-f94ba3fbbff1",
      "minLongitude": -0.6601,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.622495859317976,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T17:46:00.000+0000",
      "estimatedRestoration": "2023-04-22T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7847'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7847",
      "maxLongitude": -0.5203,
      "minLatitude": 50.8466,
      "name": "BN18 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BN18 0LT",
        "BN18 0LU",
        "BN18 0LY",
        "BN18 0NJ",
        "BN18 0NL",
        "BN18 0NN",
        "BN18 0NP",
        "BN18 0NR",
        "BN18 0NS",
        "BN18 0NT",
        "BN18 0NU",
        "BN18 0QJ",
        "BN18 0QP",
        "BN18 0QR",
        "BN18 0QS",
        "BN18 0QT",
        "BN18 0QU",
        "BN18 0QX",
        "BN18 0QY",
        "BN18 0RA",
        "BN18 0RB",
        "BN18 0RD",
        "BN18 0RE",
        "BN18 0RG",
        "BN18 0RH",
        "BN18 0RJ",
        "BN18 0RL",
        "BN18 0RN",
        "BN18 0RP",
        "BN18 0RR",
        "BN18 0RS",
        "BN18 0SA",
        "BN18 0SB",
        "BN18 0SD",
        "BN18 0SE",
        "BN18 0SF",
        "BN18 0SX",
        "BN18 0UU",
        "BN18 0UW",
        "BN18 9FD",
        "BN18 9LL",
        "BN18 9PB",
        "BN18 9RF",
        "PO20 3AA",
        "PO20 3SE",
        "RH20 1PQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6601,
              50.929
            ],
            [
              -0.5203,
              50.929
            ],
            [
              -0.5203,
              50.8466
            ],
            [
              -0.6601,
              50.8466
            ],
            [
              -0.6601,
              50.929
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "57b74c86-d8f3-45f8-8b6e-e58b5a33136c",
        "5dfb2da7-e9be-4926-ac46-625b6150383a"
      ],
      "maxLatitude": 50.929,
      "etrUpdate": true
    },
    {
      "latitude": 51.22472938891905,
      "type": "LV",
      "reference": "KU7789",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T17:30:00.000+0000",
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "527dc214-2bcb-4965-8ab0-9ecab35dea7d",
      "minLongitude": -0.6447,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6370257022216802,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T15:44:00.000+0000",
      "estimatedRestoration": "2023-04-23T03:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7789'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7789",
      "maxLongitude": -0.6242,
      "minLatitude": 51.2196,
      "name": "GU3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU3 1DG",
        "GU3 1DL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6447,
              51.2391
            ],
            [
              -0.6242,
              51.2391
            ],
            [
              -0.6242,
              51.2196
            ],
            [
              -0.6447,
              51.2196
            ],
            [
              -0.6447,
              51.2391
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2391,
      "etrUpdate": true
    },
    {
      "latitude": 50.723815624913186,
      "type": "LV",
      "reference": "KU7908",
      "etaUpdate": true,
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "120df1ac-40d6-4187-a75e-0bc3742b14e8",
      "minLongitude": -1.5872,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5816574305835502,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T19:34:00.000+0000",
      "estimatedRestoration": "2023-04-23T04:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7908'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7908",
      "maxLongitude": -1.576,
      "minLatitude": 50.7205,
      "name": "SO41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO41 0QX",
        "SO41 0QY",
        "SO41 0TB",
        "SO41 0TF",
        "SO41 0TG",
        "SO41 0UQ",
        "SO41 0XD",
        "SO41 0XF",
        "SO41 0XG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5872,
              50.7253
            ],
            [
              -1.576,
              50.7253
            ],
            [
              -1.576,
              50.7205
            ],
            [
              -1.5872,
              50.7205
            ],
            [
              -1.5872,
              50.7253
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-22T20:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7253,
      "etrUpdate": true
    },
    {
      "latitude": 50.86300452581486,
      "type": "LV",
      "reference": "KU7714",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T14:45:00.000+0000",
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6ee3258a-ef19-43de-9c6c-8e087bf442d3",
      "minLongitude": -2.1615,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1598849295146394,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T13:00:00.000+0000",
      "estimatedRestoration": "2023-04-22T21:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7714'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7714",
      "maxLongitude": -2.1588,
      "minLatitude": 50.862,
      "name": "DT11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT11 7JU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1615,
              50.8636
            ],
            [
              -2.1588,
              50.8636
            ],
            [
              -2.1588,
              50.862
            ],
            [
              -2.1615,
              50.862
            ],
            [
              -2.1615,
              50.8636
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8636,
      "etrUpdate": true
    },
    {
      "latitude": 50.69406541438068,
      "type": "LV",
      "reference": "KU7861",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T19:15:00.000+0000",
      "affectedCustomerCount": 7,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b8f043f9-9ea2-4f05-a52a-6a78b82c0a08",
      "minLongitude": -1.5355,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5291835627601813,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T19:45:00.000+0000",
      "estimatedRestoration": "2023-04-22T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7861'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7861",
      "maxLongitude": -1.525,
      "minLatitude": 50.6907,
      "name": "PO40 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO40 9SA",
        "PO40 9SS",
        "PO40 9ST",
        "PO40 9SU",
        "PO40 9SX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5355,
              50.6971
            ],
            [
              -1.525,
              50.6971
            ],
            [
              -1.525,
              50.6907
            ],
            [
              -1.5355,
              50.6907
            ],
            [
              -1.5355,
              50.6971
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.6971,
      "etrUpdate": true
    },
    {
      "latitude": 51.48274822487065,
      "type": "LV",
      "reference": "KU7916",
      "etaUpdate": true,
      "affectedCustomerCount": 52,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "cf059264-4272-4f77-9d9a-bcf604ec2b84",
      "minLongitude": -0.3963,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3937491417471898,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T19:49:00.000+0000",
      "estimatedRestoration": "2023-04-23T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7916'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7916",
      "maxLongitude": -0.3812,
      "minLatitude": 51.4799,
      "name": "TW5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "TW5 9BU",
        "TW5 9JD",
        "TW5 9JE",
        "TW5 9JH",
        "TW5 9JL",
        "TW5 9JP",
        "TW5 9JR",
        "TW5 9JW",
        "TW5 9LW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3963,
              51.4846
            ],
            [
              -0.3812,
              51.4846
            ],
            [
              -0.3812,
              51.4799
            ],
            [
              -0.3963,
              51.4799
            ],
            [
              -0.3963,
              51.4846
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4846,
      "etrUpdate": true
    },
    {
      "latitude": 50.78546412192538,
      "type": "LV",
      "reference": "KU7809",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-22T17:00:00.000+0000",
      "affectedCustomerCount": 31,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "9b6f7c40-4852-4802-b701-2abf2f1733c1",
      "minLongitude": -1.0953,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.093093071277108,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T16:22:00.000+0000",
      "estimatedRestoration": "2023-04-22T21:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7809'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7809",
      "maxLongitude": -1.0909,
      "minLatitude": 50.7842,
      "name": "PO5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO5 3JG",
        "PO5 3JN",
        "PO5 3LT",
        "PO5 3LX",
        "PO5 3NH",
        "PO5 3NL",
        "PO5 3RP"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0953,
              50.7867
            ],
            [
              -1.0909,
              50.7867
            ],
            [
              -1.0909,
              50.7842
            ],
            [
              -1.0953,
              50.7842
            ],
            [
              -1.0953,
              50.7867
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7867,
      "etrUpdate": true
    },
    {
      "latitude": 50.76603897148164,
      "type": "HV",
      "reference": "KU7922",
      "etaUpdate": true,
      "affectedCustomerCount": 338,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ed0798b9-c5c3-45e4-b2ee-eebeec4db0aa",
      "minLongitude": -1.9998,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9857545154724854,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T20:01:00.000+0000",
      "estimatedRestoration": "2023-04-22T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7922'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7922",
      "maxLongitude": -1.7816,
      "minLatitude": 50.7351,
      "name": "BH18 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH18 8AZ",
        "BH18 8BA",
        "BH18 8BB",
        "BH18 8BD",
        "BH18 8BE",
        "BH18 8BG",
        "BH18 8BH",
        "BH18 8BJ",
        "BH18 8BL",
        "BH18 8BN",
        "BH18 8BQ",
        "BH18 8BT",
        "BH18 8BZ",
        "BH18 8DB",
        "BH18 8DE",
        "BH18 8DQ",
        "BH18 8DU",
        "BH18 8EG",
        "BH18 8EJ",
        "BH18 9AR",
        "BH18 9AT",
        "BH18 9AX",
        "BH18 9AY",
        "BH18 9AZ",
        "BH18 9BE",
        "BH18 9BF",
        "BH23 1PF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.9998,
              50.7765
            ],
            [
              -1.7816,
              50.7765
            ],
            [
              -1.7816,
              50.7351
            ],
            [
              -1.9998,
              50.7351
            ],
            [
              -1.9998,
              50.7765
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-22T20:25:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7765,
      "etrUpdate": true
    },
    {
      "latitude": 56.62328706766116,
      "type": "HV",
      "reference": "KU7808",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "14db0482-5251-4ab7-a4fd-516b912d1ca0",
      "minLongitude": -6.5819,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.527055678784718,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-22T16:22:00.000+0000",
      "estimatedRestoration": "2023-04-22T22:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU7808'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU7808",
      "maxLongitude": -6.4927,
      "minLatitude": 56.6032,
      "name": "PA78 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PA78 6SY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -6.5819,
              56.6662
            ],
            [
              -6.4927,
              56.6662
            ],
            [
              -6.4927,
              56.6032
            ],
            [
              -6.5819,
              56.6032
            ],
            [
              -6.5819,
              56.6662
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.6662,
      "etrUpdate": true
    }
  ]
}
